src/configuration/services/influxdb.md

```diff
@@ -0,0 +1,80 @@
+# Influx DB(Database service)
+
+InfluxDB is a time series database optimized for high-write-volume use cases such as logs, sensor data, and real-time analytics.  It exposes an HTTP API for client interaction.
+
+See the [InfluxDB documentation](https://docs.influxdata.com/influxdb/v1.2/) for more information.
+
+## Supported versions
+
+* 1.2
+
+## Relationship
+
+The format exposed in the ``$PLATFORM_RELATIONSHIPS`` [environment variable](/development/variables.md#platformsh-provided-variables):
+
+```json
+{
+   "servicename" : [
```

Comment:
@Crell  Is this a new standard? The relationship in below section is `timedb`. If we want a generic one, it should be `relationshipname` instead of `servicename`.